_posts/2024-07-22-pyopensci-at-scipy-2024.md

```diff
@@ -0,0 +1,463 @@
+---
+layout: single
+title: "pyOpenSci a Recap of @ SciPy 2024 Meeting - Tacoma, Washington"
```

Comment:
awesome. @kierisi just a note that here you have a suggested change but the actual change is in the comment text so i can't actually "accept" the change and give you credit for it . Would you like to make the change inline so i can commit it (and the commit then is in your name! so your work gets credit). i can also modify locally - up to you!

Reply:
nope! I didn't want to assume what the title should be, so feel free to use whatever works best for you :)

Reply:
Ok got it. I suggest that you play with the how comments work and what they look like on GitHub.

This is what i see which suggests you were trying to edit the text inline. When i look at the comment, I think your intention might have been to suggest a change to the header which is why i asked you for followup. 

I think your actual intention was just to leave text feedback. it would be more clear to me then if you just left a comment rather than a text suggestion that has no actual changes to the text

<img width="556" alt="Screenshot 2024-08-02 at 10 10 08 AM" src="https://github.com/user-attachments/assets/7e0e4ba9-3176-4a34-bd1c-1dce545198f4">

does that make sense? please let me know if it doesnt. there are several comments here that look like you started to edit the text and then didn't which will prompt me to ask for more information to ensure i'm capturing the feedbac correctly.

Reply:
yep, got it! I'll be more decisive next time in whether or not I provide an inline edit, and make sure to use the corresponding feature.